src/AutoRest.CSharp/Mgmt/Generation/MgmtClientBaseWriter.cs

```diff
@@ -456,6 +505,15 @@ protected CodeWriter.CodeWriterScope WriteCommonMethod(MgmtClientOperation clien
         }
 
         protected CodeWriter.CodeWriterScope WriteCommonMethod(MethodSignature signature, FormattableString? returnDescription, bool isAsync)
+        {
+            var scope = WriteCommonMethodCommentsAndSignature(signature, returnDescription, isAsync);
+            if (This.Accessibility == "public")
+                _writer.WriteParametersValidation(signature.Parameters);
+
+            return scope;
+        }
+
+        protected CodeWriter.CodeWriterScope WriteCommonMethodCommentsAndSignature(MethodSignature signature, FormattableString? returnDescription, bool isAsync)
```

Comment:
Is any of the derived class using this new method? If not, we could make it private.

Reply:
And how about rename this to `WriteCommonMethodWithoutValidation`? Since "no validation" is the difference from the other method `WriteCommonMethod`